go/common/sgx/pcs/quote.go

```diff
@@ -456,6 +457,17 @@ func (qs *QuoteSignatureECDSA_P256) Verify(
 		return err
 	}
 
+	// Ensure PCK certification information matches the quote.
+	if !bytes.Equal(pckInfo.CPUSVN[:], qs.QEReport.CPUSVN[:]) {
+		return fmt.Errorf("pcs/quote: PCK CPUSVN: '%v', does not match QE report CPUSVN: '%v'",
+			hex.EncodeToString(pckInfo.CPUSVN[:]),
+			hex.EncodeToString(qs.QEReport.CPUSVN[:]),
+		)
+	}
+	if pckInfo.PCESVN != int32(header.PCESVN) {
```

Comment:
We should probably change `pcesvn` from `int32` to `int16` everywhere.

Reply:
Sure ok. Likely non breaking because a (malovent) non uint16 `pcesvn` would fail somewhere in the validation chain now already.